back/app/policies/initiative_policy.rb

```diff
@@ -38,7 +38,7 @@ def show?
     return true if active? && (owner? || can_moderate?)
     return false if record.review_status?
 
-    %w[draft published closed].include?(record.publication_status)
+    record.publication_status == 'published'
```

Comment:
I think in theory users should have access to their draft initiatives. But not sure it's used in practice.

Reply:
🤔

Reply:
Maybe I won't remove 'draft', just to be sure... this PR is supposed to only remove `closed` and `spam` anyway.

Reply:
Good idea.